roles/redhat_satellite6_capsule_lb/tasks/main.yml

```diff
@@ -0,0 +1,92 @@
+---
+# tasks file for satellite_lb
+
+- name: Generate LB Certs 1
+  when: inventory_hostname in groups['satellite']
+  shell: capsule-certs-generate --foreman-proxy-fqdn "{{ capsule1 }}" --certs-tar /tmp/capsule1.tar --foreman-proxy-cname "{{ lb_server }}"
+
+- name: Generate LB Certs 2
+  when: inventory_hostname in groups['satellite']
+  shell: capsule-certs-generate --foreman-proxy-fqdn "{{ capsule2 }}" --certs-tar /tmp/capsule2.tar --foreman-proxy-cname "{{ lb_server }}"
+
+- name: pull the cert from the {{ sat_server }} to the local machine
+  when: inventory_hostname in groups['satellite']
```

Comment:
you can copy directly from the sat master to the sat capsule without stopping at the host that ansible is running on https://docs.ansible.com/ansible/latest/modules/synchronize_module.html